interfaces/apparmor/template_vars.go

```diff
@@ -0,0 +1,109 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package apparmor
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	"github.com/ubuntu-core/snappy/interfaces"
+	"github.com/ubuntu-core/snappy/interfaces/dbus"
+	"github.com/ubuntu-core/snappy/snap"
+)
+
+// legacyVariablees returns text defining some apparmor variables that work
+// with legacy apparmor templates.
+//
+// The variables are expanded by apparmor parser. They are (currently):
+//  - APP_APPNAME
+//  - APP_ID_DBUS
+//  - APP_PKGNAME_DBUS
+//  - APP_PKGNAME
+//  - APP_VERSION
+//  - INSTALL_DIR
+// They can be changed but this has to match changes in template.go.
+//
+// In addition, the set of variables listed here interacts with old-security
+// interface since there the base template is provided by a particular 3rd
+// party snap, not by snappy.
```

Comment:
I was referring to old-security because those variables need to be maintained for old-security to work. Without old-security any set of variables can be used (AFAIK, perhaps I'm wrong) as long as they are changed in sync with ``template.go``.